CONTRIBUTING.md

```diff
@@ -36,6 +36,10 @@ Writing documentation
 
 Issue triaging involves organizing, prioritizing, categorizing, and labeling a project's backlog of issues and pull requests. Regularly triaging increases engagement, transparency, and productivity.
 
+#### Creating TODOs
+
+Issues can be created directly in files via TODOs. For more information, see [todos-to-issues](todos-to-issues.md).
```

Comment:
See [todos-to-issues](todos-to-issues.md) for information on how to create an Issue by adding a TODO comment directly to a file